### ISSUE TITLE
SF gets stuck at low depth during moves ~16-25 of TCEC S22 VVLTC bonus game 6

### ISSUE THREAD

**Issue by Ma-Laoshi** [official-stockfish/Stockfish#4110]
See the title. I documented the issue the best I could on Fishcooking: https://groups.google.com/g/fishcooking/c/zikyd3yDUFs
Bottom line is that recent SF's (including the one that was playing the TCEC game, and probably losing it for this very reason) get stuck in this line at a very low depth, showing no further output after the first few seconds. I verified that this happens even with only a single thread. In contrast, SF13 never exhibited this glitch when I tried it.

I have no way of knowing how common or rare this is, but when it happens it can be fatal. Feel free to combine this issue with others if they are judged to be discussing the same problem. Many thanks if one of the devs could look into this.

**Comment by vondele** [comment]
Hi this is probably relate to https://github.com/official-stockfish/Stockfish/issues/3911

**Comment by Ma-Laoshi** [comment]
"related to #3911"  <---  might very well be: as I wrote on Fishcooking, CPU/memory use all stay normal and the GUI is responsive as usual, so it's _not_ a crash. I also guessed a search explosion, which maybe is not wholly unexpected in this sharp, tactical line? But as a non-coder (so far), guessing is all I can do. Anyway, glad to see that there is more feedback here in minutes than on Fishcooking in weeks; all the best in sorting it out!
